ReactWindows/ReactNative.Net46/Modules/Launch/LauncherModule.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using System;
+using System.Reactive.Subjects;
+using static System.FormattableString;
+
+namespace ReactNative.Modules.Launch
+{
+    /// <summary>
+    /// The module responsible for activating URIs and managing the initial launch URL. 
+    /// </summary>
+    public class LauncherModule : ReactContextNativeModuleBase, ILifecycleEventListener
+    {
+        private static readonly Subject<string> s_urlSubject = new Subject<string>();
+        private static string s_activatedUrl;
+
+        private bool _initialized;
+        private IDisposable _subscription;
+
+        /// <summary>
+        /// Instantiates the <see cref="LauncherModule"/>. 
+        /// </summary>
+        /// <param name="reactContext">The React context.</param>
+        public LauncherModule(ReactContext reactContext)
+            : base(reactContext)
+        {
+        }
+
+        /// <summary>
+        /// The name of the module.
+        /// </summary>
+        public override string Name {
+            get
+            {
+                return "LinkingManager";
+            }
+        }
+
+        /// <summary>
+        /// Opens the given URL.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <param name="promise">
+        /// The promise that should be resolved after the URL is opened.
+        /// </param>
+        [ReactMethod]
+        public async void openURL(string url, IPromise promise)
```

Comment:
public methods should start with capital letters to match existing code style. (I don't like it either, but it's a C# thing.)